### PR TITLE
DOCSP-31011 Add Mongorestore Time Series Collection

### PR DESCRIPTION
**Pull request by ianf-mongodb**
## DESCRIPTION

Adding time series collection instructions to the `mongorestore` page and a example demonstrating how to restore a TS collection.

## STAGING

https://preview-mongodbianfmongodb.gatsbyjs.io/database-tools/DOCSP-31011/mongorestore/#time-series-collections

https://preview-mongodbianfmongodb.gatsbyjs.io/database-tools/DOCSP-31011/mongorestore/#restore-a-time-series-collection

## JIRA

https://jira.mongodb.org/browse/DOCSP-31011

## BUILD LOG

https://workerpool-boxgs.mongodbstitch.com/pages/job.html?collName=queue&jobId=6542866bbd8a7a9579eed6b2

## Self-Review Checklist

- [ ] Is this free of any warnings or errors in the RST?
- [ ] Is this free of spelling errors?
- [ ] Is this free of grammatical errors?
- [ ] Is this free of staging / rendering issues?
- [ ] Are all the links working?

## External Review Requirements

[What's expected of an external reviewer?](https://wiki.corp.mongodb.com/display/DE/Reviewing+Guidelines+for+the+MongoDB+Server+Documentation)

### CODE REVIEW

**Review by jmd-mongo — CHANGES_REQUESTED** [review]
Thanks for the updates, @ianf-mongodb! I've left a few suggestions for your consideration.

Thanks,
Joe

**Review by ianf-mongodb — COMMENTED** [review]
Hi @jmd-mongo, thanks for taking a look, your suggestions have been applied and this is ready for another look.

File: source/mongorestore.txt [hunk 1]
```diff
@@ -1234,6 +1243,71 @@ the filename. For example:
...
+      },
+      {
+         "metadata": { "sensorId": 5578, "type": "temperature" },
+         "timestamp": ISODate("2021-05-18T08:00:00.000Z"),
+         "temp": 11
+      }
+   ] )
+
+From your terminal, use ``mongodump`` to export the time series collection 
+to the ``dump/test`` directory. This command adds two files
```
Comment (jmd-mongo) [inline source/mongorestore.txt:1291]: [nit]

Consider one of the following:

- "This command adds two files to the directory: \``system.buckets.weather.bson\`` and \``weather.metadata.json``.
- "This commands adds \``system.buckets.weather.bson\`` and \``weather.metadata.json\`` to the directory.

**Review by jmd-mongo — COMMENTED** [review]
Thanks for the updates, @ianf-mongodb! This one LGTM mod an additional nit.

Thanks,
Joe

**Pull request merged**